### DISCUSSION TITLE
How do I update Specter Desktop?

### DISCUSSION THREAD

**Discussion by snowjam2684** [cryptoadvance/specter-desktop#909] [General]
I'm currently running v0.10.1 with Bitcoin Core on my PC and would like to update to a newer version of Specter. Whats the best/safest way to do it? I've been to the 'Releases" page but I'm not sure where to go from there.

**Comment by ben-kaufman** [comment]
Hi, should be just downloading the file of the new version based on your OS and install it regularly replacing the currently installed one for it.

**Reply by snowjam2684** [reply]
Awesome. Thank you.

**Comment by sailnomad** [comment]
Hi. I have a question. I am very interested in running my own node and taking control of my BTC. In doing research on running a node I have found they need at least 6 hours uptime per day. Is this true for Spectre Desktop as well or if I used a Pruned version do I just update it as needed. It is not feasible for me to have a guaranteed 6 hours uptime per day. Thanks for the reply.

**Reply by ben-kaufman** [reply]
Hi, that doesn't sound right, there's no such requirement at all (with or without Specter/ pruned or not) to have the node up besides when you want to actually use it. It's recommended to have it up so you don't have to wait for it to sync with the network when you really want to use it, but that doesn't matter when you choose to go online to sync

**Reply by sailnomad** [reply]
Awesome. Thanks so much for your reply. The sources I found were a year or more out of date so that probably explains the discrepancy. Thanks again for your help.
